profiles/apparmor/default-profile

```diff
@@ -0,0 +1,81 @@
+
+# NOTE: This profile is a Go template, which is necessary for the container
```

Comment:
What this file moved from somewhere? Is there anything new in it?

Reply:
No, this file didn't exist before -- it's like `profiles/seccomp/default.json` and is a plaintext version of the profile contents.